File: C/136_Single_Number.c
#include <stdio.h>
int singleNumber(int* nums, int numsSize) {
	int ans = 0;
	for (int i = 0; i < numsSize; i++){
		ans ^= nums[i];
	}
	return ans;
}

int main(){
	int temp[] = { 1, 1, 2, 3, 3, 4, 4, 5, 5 };
	printf("%d", singleNumber(temp, sizeof(temp) / sizeof(temp[0])));
	return 0;
}

File: Python/136_Single_Number.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ans = 0;
        for i in range(len(nums)):
            ans ^= nums[i]
        return ans


def test():
    s = Solution()
    temp = [0, 0, 1, 1, 2, 3, 3, 4, 4, 5, 5]
    print s.singleNumber(temp)


if __name__ == '__main__':
    test()
